src/Uno.Extensions.Navigation.UI/NavigatorExtensions.cs

```diff
@@ -66,8 +66,25 @@ public static class NavigatorExtensions
 		var gobackNavigator = await dispatcher.ExecuteAsync(async ct => region?.FindChildren(
 			child => child.Services?.GetService<INavigator>() is ControlNavigator controlNavigator &&
 				controlNavigator.CanGoBack).LastOrDefault()?.Navigator());
-		return gobackNavigator is not null ?
-			await gobackNavigator.NavigateBackAsync(sender) :
-			new NavigationResponse(Success: false);
+		return gobackNavigator;
 	}
+
+	/// <summary>
+	/// Returns value whether any of the current INavigator instances supports navigating back
+	/// </summary>
+	/// <param name="navigator">The highest level navigator</param>
+	/// <returns>bool indicating whether go back is supported</returns>
+	public static async Task<bool> CanGoBack(this INavigator navigator)
```

Comment:
Is any new documentation needed?

Reply:
yes. There currently isn't proper documentation on the various navigation extension methods